common/src/main/resources/assets/rrls/lang/zh_cn.json

```diff
@@ -0,0 +1,12 @@
+{
+  "text.autoconfig.rrls.title": "移除重新加载屏幕(RemoveReloadingScreen)",
+  "text.autoconfig.rrls.option.enabled": "隐藏重新加载屏幕",
+  "text.autoconfig.rrls.option.rgbProgress": "彩色进度条",
+  "text.autoconfig.rrls.option.rgbProgress.@Tooltip": "§c可能引起癫痫！",
```

Comment:
§c ?

Reply:
I want it to stand out because epilepsy is no joke

Reply:
Okay, I'll think about highlighting it everywhere.